src/Jira/Issues/Walker.php

```diff
@@ -117,7 +117,7 @@ class Walker implements \Iterator, \Countable
 	/**
 	 * Creates walker instance.
 	 *
-	 * @param Api          $api      API.
+	 * @param Api		  $api	  API.
```

Comment:
Darn, I wasnt exoecting the anti pattern of code alignment to be used, I guess I have to fix that up later by reverthing those changes,